scene/resources/3d/sky_material.cpp

```diff
@@ -87,7 +95,7 @@ Ref<Texture2D> ProceduralSkyMaterial::get_sky_cover() const {
 
 void ProceduralSkyMaterial::set_sky_cover_modulate(const Color &p_sky_cover_modulate) {
 	sky_cover_modulate = p_sky_cover_modulate;
-	RS::get_singleton()->material_set_param(_get_material(), "sky_cover_modulate", sky_cover_modulate);
+	RS::get_singleton()->material_set_param(_get_material(), "sky_cover_modulate", Color(sky_cover_modulate.r, sky_cover_modulate.g, sky_cover_modulate.b, sky_cover_modulate.a * sky_energy_multiplier));
```

Comment:
sky energy multiplier is multiplying alpha, not the entire thing

Reply:
That's no issue, see how `vec4 sky_cover_texture` was previously calculated in the shader vs. how it is done now. It appears equivalent to me.

Reply:
if you say so, just stood out to me at a glance

Reply:
I guess we could remove one more multiply by pre multiplying sky color modulate by its own alpha and the energy multiplier